static/app/views/settings/project/projectKeys/details/loaderSettings.tsx

```diff
@@ -0,0 +1,266 @@
+      addLoadingMessage();
+
+      const apiData: {
+        browserSdkVersion: typeof browserSdkVersion;
+        dynamicSdkLoaderOptions?: Partial<Record<DynamicSDKLoaderOption, boolean>>;
+      } = {
+        browserSdkVersion: newBrowserSDKVersion,
+      };
+
+      const shouldRestrictDynamicSdkLoaderOptions =
+        !sdkVersionSupportsPerformanceAndReplay(newBrowserSDKVersion);
+
+      if (shouldRestrictDynamicSdkLoaderOptions) {
+        // Performance & Replay are not supported before 7.x
+        const newDynamicSdkLoaderOptions = {
+          ...dynamicSDKLoaderOptions,
+          hasPerformance: false,
+          hasReplay: false,
+        };
+
+        apiData.dynamicSdkLoaderOptions = newDynamicSdkLoaderOptions;
+      }
+
+      try {
+        const response = await api.requestPromise(apiEndpoint, {
+          method: 'PUT',
+          data: apiData,
+        });
+
+        setBrowserSdkVersion(response.browserSdkVersion);
+
+        if (shouldRestrictDynamicSdkLoaderOptions) {
+          setDynamicSDKLoaderOptions(response.dynamicSdkLoaderOptions);
+        }
+
+        addSuccessMessage(t('Successfully updated SDK version'));
+      } catch (error) {
+        const message = t('Unable to updated SDK version');
+        handleXhrErrorResponse(message)(error);
+        addErrorMessage(message);
+      }
+    },
+    [
+      api,
+      apiEndpoint,
+      setBrowserSdkVersion,
+      setDynamicSDKLoaderOptions,
+      dynamicSDKLoaderOptions,
+    ]
+  );
+
+  return (
+    <Access access={['project:write']}>
+      {({hasAccess}) => (
+        <Fragment>
+          <FieldGroup
+            help={tct(
+              'Copy this script into your website to setup your JavaScript SDK without any additional configuration. [link]',
+              {
+                link: (
+                  <ExternalLink href="https://docs.sentry.io/platforms/javascript/install/lazy-load-sentry/">
+                    What does the script provide?
+                  </ExternalLink>
+                ),
+              }
+            )}
+            inline={false}
+            flexibleControlStateSize
+          >
+            <TextCopyInput>
+              {`<script src='${loaderLink}' crossorigin="anonymous"></script>`}
+            </TextCopyInput>
+
+            <FieldHelp style={{paddingTop: space(1)}}>
+              {t('Note that it can take a few minutes until changed options are live.')}
+            </FieldHelp>
+          </FieldGroup>
+
+          <SelectField
+            name="browserSdkVersion"
+            label={t('SDK Version')}
+            options={
+              projectKey.browserSdk
+                ? projectKey.browserSdk.choices.map(([value, label]) => ({
+                    value,
+                    label,
+                  }))
+                : []
+            }
+            value={browserSdkVersion}
+            onChange={value => handleUpdateBrowserSDKVersion(value)}
+            placeholder={t('4.x')}
```

Comment:
I believe we don't need to translate `4.x`. do we?
```suggestion
            placeholder="4.x"
```

Reply:
True, actually probably should update this to show "7.x" as a placeholder instead 😅